EIPS/eip-1108.md

```diff
@@ -76,23 +103,16 @@ If zk-SNARK verification costs are decreased, these solutions can be deployed fo
 
 This EIP does not require Parity or Geth to deploy new cryptographic libraries, as fast bn128 algorithms have already been integrated into these clients. This goal of proposing this EIP for Istanbul, is to supplement [EIP-1829](https://github.com/ethereum/EIPs/blob/master/EIPS/eip-1829.md) (arithmetic over generic elliptic curves), providing an immediate solution to the pressing problem of expensive cryptography, while more advanced solutions are developed, defined and deployed.
 
-## Specification
 
-Following is a table with the current gas cost and new gas cost:
-
-| Contract      | Address   | Current Gas Cost               | Updated Gas Cost    |
-| ------------- | --------- | -----------------------------  | ------------------- |
-| `ECADD`       | `0x06`    | 500<sup>[1]</sup>              | 150                 |
-| `ECMUL`       | `0x07`    | 40 000<sup>[1]</sup>           | 6 000               |
-| Pairing check | `0x08`    | 80 000 * k + 100 000<sup>[2]</sup>| 34 000 * k + 45 000    |
+## Test Cases  
 
-The gas costs for `ECADD` and `ECMUL` are updates to the costs listed in
-EIP-196, while the gas costs for the pairing check are updates to the cost
-listed in EIP-197. Updated gas costs have been adjusted to the less performant 
-client which is Parity, according to benchmarks<sup>[3]</sup>. The updated gas costs are scaled relative to the `ecrecover` precompile.
+As no underlying algorithms are not being changed, there are no additional test cases to specify.  
```

Comment:
“As no underlying algorithms are being* changed.”, or we're changing ALL OF THE THINGS :)

Reply:
> “As no underlying algorithms are being* changed.”, or we're changing ALL OF THE THINGS :)

Came here to point that language out =)

Reply:
I'll push a PR that fixes it in a few.